bin/key_server.js

```diff
@@ -110,6 +110,7 @@ function main() {
         customs.close()
         mailer.stop()
         database.close()
+        process.exit() //XXX: because of openid dep ಠ_ಠ
```

Comment:
Ah non-closed connections strike back!

IMHO We should file an issue on https://github.com/havard/node-openid/issues